stock_picking_volume/tests/test_stock_picking_volume.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2023 ACSONE SA/NV
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestStockPickingVolume(TransactionCase):
+    def setUp(self):
+        super().setUp()
```

Comment:
@mt-software-de Better to do a SavepointCase with a setUpClass to avoid recreating it between tests.